DevEdu.API/DevEdu.API/Controllers/GroupController.cs

```diff
@@ -65,44 +65,38 @@ public void ChangeGroupStatus(int groupId, int statusId)
         [HttpPost("{groupId}/lesson/{lessonId}")]
         public void AddGroupLessonReference(int groupId, int lessonId)
         {
-            _groupRepository.AddGroupLesson(groupId,  lessonId);
+            //_groupService.AddGroupLesson(groupId,  lessonId);
```

Comment:
please uncomment such rows
otherwise it's not possible to test these endpoints until someone other will change it to service